.travis.yml

```diff
@@ -18,4 +18,14 @@ matrix:
   include:
     - rvm: jruby
       env: JRUBY_OPTS='--2.0'
+    - rvm: 2.2
+      env: RSPEC='3-0-maintenance'
+    - rvm: 2.2
+      env: RSPEC='3-1-maintenance'
+    - rvm: 2.2
+      env: RSPEC='3-2-maintenance'
+    - rvm: 2.2
+      env: RSPEC='3-3-maintenance'
+    - rvm: 2.2
+      env: RSPEC='3-4-maintenance'
```

Comment:
Yeah I'm not entirely sure this approach will even work at the moment, I just want to see what travis does :)